plugin/package.json

```diff
@@ -126,13 +125,6 @@
     "webpack-cli": "^4.9.1",
     "webpack-dev-server": "^4.5.0"
   },
-  "resolutions": {
```

Comment:
I have reviewed all dependency resolutions and found them to be outdated. The previous versions of these libraries no longer exist to be overridden in the `yarn.lock` file. I haven't yet checked the Kiali repository, but it's likely that this change can also be applied there.